plugins/woocommerce-blocks/tests/e2e/.eslintrc.js

```diff
@@ -23,6 +23,10 @@ const config = {
 		'playwright/prefer-web-first-assertions': 'error',
 		'playwright/valid-expect': 'error',
 		'rulesdir/no-raw-playwright-test-import': 'error',
+		'playwright/no-hooks': [
+			'error',
+			{ allow: [ 'afterEach', 'beforeEach' ] },
```

Comment:
We should only allow the `beforeEach` hook since we're clearing the db within the `page` fixture:

```suggestion
			{ allow: [ 'beforeEach' ] },
```